csharp/ql/integration-tests/posix-only/standalone_dependencies_nuget_no_mono/Assemblies.expected

```diff
@@ -1,242 +1,236 @@
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.Http.SelfHost.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.Http.WebHost.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.Http.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.Mobile.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.Mvc.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.Razor.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.RegularExpressions.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.Routing.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.Services.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.WebPages.Deployment.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.WebPages.Razor.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.WebPages.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Web.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Windows.Forms.DataVisualization.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Windows.Forms.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Windows.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Workflow.Activities.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Workflow.ComponentModel.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Workflow.Runtime.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Xaml.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Xml.Linq.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Xml.Serialization.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.Xml.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/System.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/WebMatrix.Data.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/WindowsBase.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/cscompmgd.dll |
-| /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/4.8-api/mscorlib.dll |
 | /legacypackages/Newtonsoft.Json.6.0.4/lib/portable-net45+wp80+win8+wpa81/Newtonsoft.Json.dll |
+| /missingpackages/microsoft.netframework.referenceassemblies.net481/1.0.3/build/.NETFramework/v4.8.1/Accessibility.dll |
```

Comment:
Why do these go into the root and not the DB scratch dir?

Reply:
They are in the scratch dir, we substring the paths in the test: https://github.com/github/codeql/pull/15600/files#diff-2f4399343179c5277fff6c12b814c1c1778b27544f0cdff6705fa21392320987

Reply:
I should probably prepend `[...]` to the paths. I'll do it in a followup PR, because it applies to some other integration tests too.